Sources/Commands/SwiftPackageTool.swift

```diff
@@ -175,6 +175,8 @@ public class SwiftPackageTool: SwiftTool<PackageToolOptions> {
             describe(graph.rootPackages[0].underlyingPackage, in: options.describeMode, on: stdoutStream)
 
         case .dumpPackage:
+            options.shouldMuteOutput = !options.dumpPackageOptions.includeToolOutput
+
             let graph = try loadPackageGraph()
             let manifest = graph.rootPackages[0].manifest
```

Comment:
Given that we're just dumping the root manifest, we shouldn't even use loadPackageGraph method to get the manifest. This will automatically avoid the problem of muting the resolution output (as there will be no resolution).